### ISSUE TITLE
cornzyblack_L02E14

### ISSUE THREAD

**Issue by cornzyblack** [mikeudacity/robo-pace-betatest#109]
When i first saw challenge 3:T3 I thought that the file named unixCommands.txt was going to contain a list of unix commands and examples (as i am not familiar with Unix), however it didn't. I had to use google to search for Unix Commands. It would be nice if basic Unix commands were provided, either as a link in the episode or as a topic in a previous episode and not only when the Learner fails the Quiz.

**Comment by cangelogamboa** [comment]
Good point, let me think about that. Thank you for bringing it to my attention.
